ui/apps/platform/src/Containers/Collections/RuleSelector/RuleSelector.tsx

```diff
@@ -15,10 +14,25 @@ import {
 import ByNameSelector from './ByNameSelector';
 import ByLabelSelector from './ByLabelSelector';
 
+import './RuleSelector.css';
+
 function isRuleSelectorOption(value: string): value is RuleSelectorOption {
     return selectorOptions.includes(value as RuleSelectorOption);
 }
 
+const placeholders = {
+    ByName: {
+        Deployment: 'nginx-deployment',
```

Comment:
Was there any discussion about placeholders less ambiguous with an actual names for people who might not be at their best attention?

Like `'deployment-name'` and so on.

Reply:
We went with the placeholders that were available in the official k8s documentation when available, but we can bring it up at the sync and see if other have better suggestions.